.github/workflows/amd-offline-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: AMD Offline Tests
```

Comment:
This workflow is a temporary solution, to demonstrate that off-line tests are working.
This solution should be changes in future.

The problem is: docker does not have access to target hardware and it uses pytorch without GPU support. This could  induce some unexpected effects.